### ISSUE TITLE
Behavior score - Overly sensitive in Turbo mode?

### ISSUE THREAD

**Issue by gitguddota2** [ValveSoftware/Dota2-Gameplay#13369]
### Behavior Score/Conduct Summary excessively brutal drops in score

TLDR: I believe there must be some sort of integer error affecting the conduct summary and behavior score system. I provided data documenting my personal conduct summary numbers from Steam's Personal Game Data page.

Preamble: 
I have been playing Turbo mode in order to try to raise my behavior score. This failed miserably as evidenced by the current behavior score of 1. I have been punished by the Overwatch system multiple times with bans and low priority. I do not believe I have been acting in a way in game to warrant, especially in turbo games where it's supposed to be a casual mode for fun, and yet the amount of reports is insanely high from the players. Some of these conduct summaries were generated on the same day, either by actually playing 15 games in a day, or it auto-triggered and decreased score and gave me low priority. Again, I have been treating turbo games as a job, working to raise the score and have submitted requests to steam support about this. They have directed me to open a ticket here.

Request:
Please look at the data and see if there are any outstanding irregularities which would contribute to such a rapid decrease in score. 

I have a few questions to interpret from the data.
Why is there a negative integer? -34 and -67 reports
Why am i losing more score between summaries where more games were marked "positive" than not?
Why do i not gain score where there are 2 matches reported by 6 players compared to gaining score 27 reports by 12 players? These summaries are classified as Periodic and not Excessive Reports.
Is it statistically relevant to have so many commends and so many reports in the same conduct summary?
It has been a long-standing theory of mine that commends do not affect score, but if they are somehow not being counted then.
Are communication abuse reports being counted when below 3000 score and pings and voicelines are rate-limited to once every 60 seconds? Why? What about below 6000 score when you can use voicelines but not text chat? Is this counted as negative communication? What about below 7000 score, are multiple pings with a communication abuse report being triggered as a true positive? Why in God's name if yes?

I have included 4 screenshots that valve can decipher the Unix time of where patch 7.33 begins and where the change was made to the behavior score. It is my theory the system was reverted a few weeks after implementation, because people were. During this period, I personally had no complaints and thought my game quality was acceptable. I reached 5000 MMR for the first time during this period. I am currently banned from Ranked.

The screenshots are of the personal game data accessible on https://steamcommunity.com/profiles/76561198384096495/gcpd/570/?category=Account&tab=MatchPlayerReportIncoming 
(For anyone reading this who would like to see their own game data, please replace my Steam user number in the link with your own. Follow Account -> Incoming Match Player Report.)
The column dates of course are organized from most recent to latest. 

I will be following the thread closely, please do add your thoughts if you have any comments about my data. Thank you.

### Example Match ID: 7423620413 

In the last turbo game I played before I was issued an automated 24 hour ban + at this time unknown number of low priority games, and a 1299 drop in behavior score to 1. I was matched with a party of 4, who took it upon themselves to grief and scapegoat me, as well as report me. I reported them and all 4 heros showed up as "action taken". It was easy to tell since I didn't report any other players. At 12 minutes Magnus wrote in the chat he was done. At around 15-16 min Magnus wasted RP in our fountain, used refresher, and wasted RP again.  The game was over at 17 min, but I believe we could have won if the party of 4 did not give up.
my dotabuff: https://www.dotabuff.com/players/423830767

### Screenshots

<img width="1306" alt="Screen Shot 2023-11-06 at 8 09 09 PM" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/150096380/27cd95d6-37d1-4813-964a-031ea61e30bc">

<img width="1306" alt="Screen Shot 2023-11-06 at 8 09 37 PM" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/150096380/aab1b797-ef5f-4db8-9247-b693e325c371">

<img width="1306" alt="Screen Shot 2023-11-06 at 8 09 51 PM" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/150096380/c1b76fe8-12c5-400d-b793-32bef4e6cd67">

<img width="1306" alt="Screen Shot 2023-11-06 at 8 13 28 PM" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/150096380/a78ba08c-fa51-4c1b-82e0-159aff78200a">

**Comment by Redo02** [comment]
same. we have nothin to lose now
just REPORT EVERY ONE IN MATCH MAKE ALL player to 1 score so they will notice this shit

**Comment by marcelmaster** [comment]
#13340
same

**Comment by BledNavalny** [comment]
Its not Valve, its Calve.
My suggest - revert it back to august WHEN WE PLAYED GREAT FOR 12 YEARS and
DONT.          TOUCH.          ANYTHING

Im not even talking about poor compendium, about which i dont care and 3 mln prize pool, 

JUST.     DONT.     TOUCH.     ANYTHING

**Comment by gitguddota2** [comment]
> same. we have nothin to lose now just REPORT EVERY ONE IN MATCH MAKE ALL player to 1 score so they will notice this shit

No, I'm sorry but this is not a good solution. False reports and people who were detect
